tutormfe/plugin.py

```diff
@@ -24,55 +24,65 @@
 }
 
 
-# If the package version suffix is set (for instance, in the nightly branch) use the "heads" Github refs API endpoint by default.
-def gh_refs_path() -> str:
-    return "heads" if __version_suffix__ else "tags"
+def get_github_refs_path(owner: str, name: str) -> str:
+    """
+    Generate a URL to access refs in heads (nightly) or tags (stable) via Github API.
+    Args:
+        owner (str): The name of the Github repository owner, as a string in 'owner' format.
+        name (str): The name of the Github repository, as a string in 'repo' format.
+
+    Returns:
+        str: A string URL to the Github API, pointing to heads if version_suffix is set, tags otherwise.
+
+    """
+    url = f"https://api.github.com/repos/{owner}/{name}/git/refs/"
+    return url + "heads" if __version_suffix__ else url + "tags"
```

Comment:
```
return f"https://api.github.com/repos/{owner}/{name}/git/refs/{'heads' if __version_suffix__ else 'tags'}"
```

Reply:
Also, let's remove the `owner="openedx"` argument from this function.

Reply:
sure.